remotespark/livyclientlib/pandasscalalivyclient.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) 2015  aggftw@gmail.com
+# Distributed under the terms of the Modified BSD License.
+
+import pandas as pd
+import json
+import re
+
+from .log import Log
+from .livyclient import LivyClient
+
+
+class PandasScalaLivyClient(LivyClient):
```

Comment:
It looks like there's a lot of duplicated code between the two pandas clients.  Can we merge them into a single class (the same way we only have a single LivyClient) or at least extract some of their shared functionality into a common place?